### ISSUE TITLE
Connected with result code [5]

### ISSUE THREAD

**Issue by dekiesel** [mczerski/MySensorsSerial2MQTT#1]
I am getting this when running the script. Is this because I can't/haven't set a username/pw to the mqtt broker?

**Comment by mczerski** [comment]
this script does not support authentication, so if Your mqtt broker require authentication it won't work.

**Comment by dekiesel** [comment]
Is there a technical reason it doesn't support authentication? If paho-mqtt supports it I might create a pull request after I've done the changes. 

Sent from Android
________________________________

**Comment by mczerski** [comment]
no, there is no technical reason. I just didn't implemented it. Feel free to create a pull request.
